devtools/create_project/msvc10/create_project.vcxproj

```diff
@@ -59,10 +59,8 @@
     </Link>
     <PostBuildEvent>
       <Command>@echo off
-xcopy /Y "$(TargetPath)" "$(SolutionDir)\..\..\..\dists\msvc10\"
-xcopy /Y "$(TargetPath)" "$(SolutionDir)\..\..\..\dists\msvc9\"
-xcopy /Y "$(TargetPath)" "$(SolutionDir)\..\..\..\dists\codeblocks\"
-xcopy /Y "$(TargetPath)" "$(SolutionDir)\..\..\..\dists\iphone\"</Command>
```

Comment:
Why are we copying codeblocks and iphone to begin with?

Reply:
Good question, it was already like that before I got here. :) Theoretically, one could compile create_project in MSVC and later decide to generate a CodeBlocks project with it, but it seems unlikely.
Maybe it would make more sense for the MSVC project to only copy to the MSVC target?